packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -564,16 +564,11 @@ class HotRunner extends ResidentRunner {
     if (benchmarkMode) {
       final List<Future<void>> isolateNotifications = <Future<void>>[];
       for (final FlutterDevice device in flutterDevices) {
+        await device.vmService.streamListen('Isolate');
         for (final FlutterView view in device.views) {
           isolateNotifications.add(
-            view.owner.vm.vmService.onIsolateEvent
-              .then((Stream<ServiceEvent> serviceEvents) async {
-              await for (final ServiceEvent serviceEvent in serviceEvents) {
-                if (serviceEvent.owner.name.contains('_spawn')
```

Comment:
TODO: verify spawn check is not necessary

Reply:
@zanderso I've updated the logic here, the previous event notifications we were using are technically private and aren't exposed any more. However, the entire purpose was just to guesstimate when the new isolate was ready, which I can still do by listening for an extension to be registered.